### PR TITLE
Add reference from users of the app

### PR DESCRIPTION
**Pull request by fmutlu**
This PR allows an user to : 

- Easily add a reference if the latter uses the app

### CODE REVIEW

**Comment by KnightEther** [comment]
Do you bind the reference with the user or do you just get the user informations when adding the reference?

**Comment by fmutlu** [comment]
> 
> 
> Do you bind the reference with the user or do you just get the user informations when adding the reference?

Only the profile picture was binded but I changed this in my last commit. Now all the informations are binded.

File: app/src/main/java/ch/epfl/sweng/thegramginers/Profile/ProfileFragment.java [hunk 1]
```diff
@@ -163,9 +156,26 @@ private void cleanReferences(View view) {
...
+                String referenceHeadline;
+                String referenceWebsite;
+                String referenceEmail;
+                if(currentReference.getId().contains("-")){
+                    referenceName = currentReference.getName();
+                    referenceEmail = currentReference.getEmail();
+                    referenceWebsite = currentReference.getWebsite();
+                    referenceHeadline = currentReference.getHeadline();
+                } else {
+                    referenceName = User.fromId(currentReference.getId()).getName();
```
Comment (KnightEther) [inline app/src/main/java/ch/epfl/sweng/thegramginers/Profile/ProfileFragment.java]: You don't know if the user has already been loaded, you need to check if he is synchronized and if not add a propertyChangedListener that wait for it.

Comment (fmutlu) [reply app/src/main/java/ch/epfl/sweng/thegramginers/Profile/ProfileFragment.java]: I implemented it in my last commit. Thank you for the review

**Comment by KnightEther** [comment]
The last commits are good for me. Good jobs!
Moreover it cleans ProfileFragment which is awesome 👍

**Pull request merged**